.env.sample

```diff
@@ -0,0 +1,3 @@
+DATABASE_URL=
+DEV_DATABASE_URL=postgres://kuvsqczu:g_zoAu9DRRF3kINngz1sT8UAe-d26Dyv@raja.db.elephantsql.com:5432/kuvsqczu
```

Comment:
@chibytez This is fine. I hope the values here are not the actual values for development